src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/EditAndSyncFeatures/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Name": "Edit and sync features",
+  "SampleName": "EditAndSyncFeatures",
+  "Description": "This sample demonstrates how to synchronize offline edits with a feature service.",
+  "Instructions": "1. Pan and zoom to the area you would like to download features for, ensuring that all features are within the rectangle.\n2. Tap the 'generate' button. This will start the process of generating the offline geodatabase.\n3. Tap on a feature within the area of the generated geodatabase. Then tap on the screen (anywhere within the range of the local geodatabase) to move the feature to that location.\n4. Tap the 'Sync Geodatabase' button to synchronize the changes back to the feature service.\n\n Note that the basemap for this sample is downloaded from ArcGIS Online automatically.",
```

Comment:
Explicitly tell the user in the instructions that they need to tap on a "Point" feature to move. They cannot click and move polylines and polygons!